include/mbedtls/ssl.h

```diff
@@ -110,6 +110,7 @@
 #define MBEDTLS_ERR_SSL_UNEXPECTED_RECORD                 -0x6700  /**< Record header looks valid but is not expected. */
 #define MBEDTLS_ERR_SSL_NON_FATAL                         -0x6680  /**< The alert message received indicates a non-fatal error. */
 #define MBEDTLS_ERR_SSL_INVALID_VERIFY_HASH               -0x6600  /**< Couldn't set the hash for verifying CertificateVerify */
+#define MBEDTLS_ERR_SSL_BAD_CONFIG                        -0x6580  /**< Invalid value in SSL config */
```

Comment:
Use the same numerical values in backports. This is more important than keeping values consecutive. It lets people consult documentation from another version, or use `strerror` from another version, without getting incorrect information.

Reply:
Done

Reply:
Done